### PR TITLE
Adds some basic sanity to the plumbing bottler

### PR DESCRIPTION
**Pull request by optimumtact**
## About The Pull Request
Spotted a runtime from a null output location, this should really sanity check these.

closes #62820

### CODE REVIEW

File: code/modules/plumbing/plumbers/bottler.dm [hunk 1]
```diff
@@ -24,6 +26,8 @@
 /obj/machinery/plumbing/bottler/examine(mob/user)
 	. = ..()
 	. += span_notice("A small screen indicates that it will fill for [wanted_amount]u.")
+	if(!valid_output_configuration)
+		. += span_warning("Output location error condition warning is flashing on the display")
```
Comment (MrDoomBringer) [inline code/modules/plumbing/plumbers/bottler.dm:30]:
```suggestion
		. += span_warning("A flashing notification on the screen reads: \"Output location error!\"")
```

File: code/modules/plumbing/plumbers/bottler.dm [hunk 2]
```diff
@@ -52,15 +56,28 @@
...
+	//If by some miracle
+	if( ( !valid_output_configuration ) && ( goodspot != null && inputspot != null && badspot != null ) )
+		valid_output_configuration = TRUE
+		begin_processing()
+
 ///changing input ammount with a window
 /obj/machinery/plumbing/bottler/interact(mob/user)
 	. = ..()
+	if(!valid_output_configuration)
+		to_chat(user, span_warning("Output location error condition warning is flashing on the display"))
```
Comment (MrDoomBringer) [inline code/modules/plumbing/plumbers/bottler.dm:68]:
```suggestion
		to_chat(user, span_warning("A flashing notification on the screen reads: \"Output location error!\""))
```

File: code/modules/plumbing/plumbers/bottler.dm [hunk 3]
```diff
@@ -52,15 +56,28 @@
...
+		return .
```
Comment (MrDoomBringer) [inline code/modules/plumbing/plumbers/bottler.dm:69]:
```suggestion
		return
```

**Review by MrDoomBringer — CHANGES_REQUESTED** [review]

File: code/modules/plumbing/plumbers/bottler.dm (hunk 1 above)
Comment (optimumtact) [reply code/modules/plumbing/plumbers/bottler.dm:30]: no your version is bad

File: code/modules/plumbing/plumbers/bottler.dm (hunk 3 above)
Comment (optimumtact) [reply code/modules/plumbing/plumbers/bottler.dm:69]: better to be explicit than implicit

Comment (Mothblocks) [reply code/modules/plumbing/plumbers/bottler.dm:69]: I hate `.` and always do stuff like this

**Pull request merged**